include/odp/api/spec/ipsec.h

```diff
@@ -1148,6 +1148,14 @@ typedef enum odp_ipsec_status_id_t {
 	 */
 	ODP_IPSEC_STATUS_SA_DISABLE = 0,
 
+	/** Received dummy packet with NextHeader set to NoNextHeader
+	 *
+	 *  Following status event (odp_ipsec_status_t) fields have valid
+	 *  content, other fields must be ignored:
+	 *  - sa:       The SA that caused the warning
+	 */
+	ODP_IPSEC_STATUS_NONH,
+
```

Comment:
If this were an `odp_ipsec_packet_result_t` bit rather than a status event all input packets (synchronous, asynchronous, and inline) cases could be handled the same without the TBD. Why use status events for these packets?

Reply:
Because there might be no packet associated with such event. Just information that there was TFC packet